Pod/Classes/WPMediaCollectionDataSource.h

```diff
@@ -225,7 +225,7 @@ typedef int32_t WPMediaRequestID;
  *
  *  @return an object implementing the WPMediaAsset protocol
  */
-- (id<WPMediaAsset>)mediaAtIndex:(NSInteger)index;
+- (nonnull id<WPMediaAsset>)mediaAtIndex:(NSInteger)index;
```

Comment:
Same question as for `groupAtIndex:`?

Reply:
I was thinking if you use an index out of bounds to throw an exception, like the system array.